Dockerfile

```diff
@@ -38,5 +40,8 @@ COPY ./kvrocks.conf  ./conf/
 RUN sed -i -e 's%dir /tmp/kvrocks%dir /var/lib/kvrocks%g' ./conf/kvrocks.conf
 VOLUME /var/lib/kvrocks
 
+COPY ./LICENSE ./NOTICE ./DISCLAIMER .
+COPY ./licenses ./licenses
```

Comment:
Step 16/19 : COPY ./LICENSE ./NOTICE ./DISCLAIMER .
When using COPY with more than one source file, the destination must be a directory and end with a /

Reply:
Please verify locally first :)

Reply:
It is weird. Docker in my computer does not have this error : (

Reply:
done